packages/cli-plugin-ionic-angular/src/generate.ts

```diff
@@ -37,21 +37,25 @@ export async function generate(args: CommandHookArgs): Promise<string[]> {
   const context = AppScripts.generateContext();
 
   const [ type, name ] = args.inputs;
+  const includeNgModule = args.options.module;
+  const includePageConstants = args.options.constants;
+
+  // args.env.log.msg(() => `${includePageConstants}`);
 
   switch (type) {
     case 'page':
-      await AppScripts.processPageRequest(context, name);
+      await AppScripts.processPageRequest(context, name, includeNgModule, includePageConstants);
```

Comment:
Maybe move the options into an object? I can imagine there being a _lot_ more than 2 options for generators. We probably don't want 10+ positional args, especially cross-library.